swagger.yaml

```diff
@@ -198,13 +198,13 @@ paths:
         - in: path
           name: id
           description: "Unique id of policy"
-          type: integer
+          type: string
           required: true
       responses:
         200:
           description: "Successfully updated a policy for a given id"
           schema:
-            $ref: "#/definitions/Policy"
+            $ref: "#/definitions/NewPolicy"
```

Comment:
did you mean to put this `NewPolicy` as the parameter for `POST /policies` ? 
That's where it makes sense to me, as I'd assume the user shouldn't have to provide an `id` in the body

Reply:
Great spot